fastly-compute/geoip/README.md

```diff
@@ -0,0 +1,11 @@
+# Empty Starter Kit for JavaScript
```

Comment:
Update this doc with some helpful information!

I'd suggest at a minimum:
  - Changing the title
  - Giving a brief description of what this service does
  - Adding some notes on how to work on it locally (npm install, commands to start the server, etc)

Reply:
Definitely makes sense, I'll be sure to work this read.me over.